specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/inference.json

```diff
@@ -0,0 +1,1249 @@
+                  "readOnly": true
+                },
+                "codeAssetId": {
+                  "type": "string",
+                  "description": "ResourceId of the Code Assets i.e. startup code.",
+                  "readOnly": true
+                },
+                "vnet": {
+                  "$ref": "#/definitions/VirtualNetworkProfile",
+                  "description": "Description of the Virtual Network and subnet for outbound calls from this deployment.",
+                  "readOnly": true
+                }
+              }
+            }
+          }
+        }
+      ]
+    },
+    "InferenceDeploymentCreateRequest": {
+      "description": "The request to create an Inference Deployment.",
+      "type": "object",
+      "required": [
+        "modelIds",
+        "skuSize",
+        "weight"
+      ],
+      "properties": {
+        "tags": {
+          "description": "Contains resource tags defined as key/value pairs.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "modelIds": {
+          "description": "The list of model resource Ids.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "skuSize": {
+          "description": "sku size for the deployment",
+          "type": "string"
+        },
+        "weight": {
+          "description": "weight of the deployment for load balancing.",
+          "type": "integer",
+          "minimum": 0,
+          "exclusiveMinimum": true
+        },
+        "scaleSettings": {
+          "description": "Scale settings for this deployment",
+          "$ref": "#/definitions/SkuScaleSettings"
+        },
+        "environmentConfigId": {
+          "type": "string",
+          "description": "Resource Id of the Environment Configuration artifact."
+        },
+        "codeAssetId": {
+          "type": "string",
+          "description": "Resource Id of the Code Assets i.e. startup code."
+        },
+        "vnet": {
+          "description": "Info of the customer's custom Virtual Network",
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "ResourceId of the Virtual Network."
+            },
+            "subnetName": {
+              "type": "string",
+              "description": "Name of the Subnet in the Virtual Network."
+            }
+          }
+        }
+      }
+    },
+    "SkuScaleSettings": {
+      "description": "Description of the Deployment scale options.",
+      "type": "object",
+      "required": [
+        "minimum",
+        "maximum"
+      ],
+      "properties": {
+        "minimum": {
+          "format": "int32",
+          "description": "Minimum number of instances for this SKU.",
+          "type": "integer",
+          "minimum": 1,
+          "exclusiveMinimum": true
```

Comment:
what this?

Reply:
need to delete this as it excludes 1 as a possible value to enter.